include/fty-log/fty_logger.h

```diff
@@ -141,7 +141,19 @@ inline std::string format(const std::string& str)
 template<typename... Args>
 inline std::string format(const std::string& str, const Args&... args)
 {
-    return fmt::format(str, args...);
+    //fmt format can thow exception
+    //In debug mode we want to take benefits of those exception to solve the formating issue
+    //However that's not a good idea for production
+    std::string returnStr;
+    try {
+        returnStr = fmt::format(str, args...);
+    } catch(std::exception& e) {
+#ifdef DEBUG
+        throw;
+#endif
+        returnStr = str + " /!\ A log serialization error happened: " + e.what();
+    }
+    return returnStr;
```

Comment:
Return string directly could 
```suggestion
    try {
        return fmt::format(str, args...);
    } catch(const std::exception& e) {
#ifdef DEBUG
        throw;
#endif
    }
    return str + " /!\\ A log serialization error happened: " + e.what();
```
fixed return string, const exception, and \ - is a escape symbol.